src/pages/app-our-work-single/app-our-work-single.scss

```diff
@@ -0,0 +1,424 @@
+        font-size: 28px;
+      }
+
+      @include media-breakpoint-down(sm) {
+        text-align: left;
+        padding: 0 28px;
+      }
+    }
+
+    p {
+      margin-top: 30px;
+      color: #292a2d;
+      font-family: 'Muli';
+      font-size: 18px;
+      font-weight: 400;
+      line-height: 36px;
+      text-align: left;
+      margin-bottom: 30px;
+      width: 80%;
+
+      @include media-breakpoint-down(md) {
+        font-size: 16px;
+        width: initial;
+      }
+    }
+
+    .project-content-icons {
+      @include media-breakpoint-down(sm) {
+        display: flex;
+        justify-content: center;
+        margin-top: 60px;
+      }
+
+      img {
+        margin: 10px 40px 10px 0;
+
+        @include media-breakpoint-down(sm) {
+          margin: 10px 20px 10px 0;
+        }
+      }
+    }
+
+    .store-buttons {
+      margin-top: 16px;
+
+      @include media-breakpoint-down(sm) {
+        display: flex;
+        justify-content: center;
+        margin-top: 140px;
+      }
+
+      a {
+        margin: 10px 20px 10px 0;
+      }
+    }
+  }
+
+  section.key-features {
+    margin-top: 80px;
+
+    .key-features-text {
+      background-color: #292a2d;
+      padding-bottom: 36px;
+      padding-top: 60px;
+
+      h2 {
+        color: #ffffff;
+        font-family: 'Muli';
+        font-size: 48px;
+        font-weight: 600;
+        letter-spacing: 1.6px;
+        line-height: 58px;
+        text-align: center;
+
+        @include media-breakpoint-down(sm) {
+          font-size: 28px;
+          text-align: left;
+        }
+      }
+
+      p {
+        margin-top: 35px;
+        color: #ffffff;
+        font-family: 'Muli';
+        font-size: 16px;
+        font-weight: 400;
+        line-height: 32px;
+        text-align: center;
+        width: 715px;
+        margin-left: auto;
+        margin-right: auto;
+
+        @include media-breakpoint-down(sm) {
+          width: initial;
+          text-align: left;
+        }
+      }
+    }
+
+    .key-features-imgs {
+      display: flex;
+      justify-content: center;
+      padding-top: 55px;
+      padding-bottom: 60px;
+
+      &.mobilemeasures {
+        background-image: url('/assets/apps/mobilemeasures/background-graphic.png');
+        background-size: cover;
+      }
+
+      &.loudcloud {
+        background-image: url('/assets/home-graphic-loudcloud-pattern.jpg');
+        background-size: cover;
+      }
+
+      &.voyage {
+        background-image: url('/assets/home-graphic-voyage-pattern.jpg');
+        background-size: cover;
+      }
+
+      img {
+        width: 280px;
+        height: 500px;
+        margin: 10px 20px;
+
+        @include media-breakpoint-down(md) {
+          width: 180px;
+          height: 320px;
+        }
+      }
+    }
+  }
+
+  section.more-projects {
+    margin-top: 120px;
+
+    @include media-breakpoint-down(sm) {
+      margin-top: 40px;
+    }
+
+    h2 {
+      color: #292a2d;
```

Comment:
Let's use the global variable name for this, I believe we have it. If not, let's create it.